haptools/data/haplotypes.py

```diff
@@ -469,6 +488,35 @@ def transform(self, genotypes: GenotypesRefAlt) -> npt.NDArray[bool]:
         # and then just AND them together
         return np.all(allele_arr == gts.data, axis=1)
 
+    def __lt__(self, other):
```

Comment:
```suggestion
    def __lt__(self, other: Haplotype):
```
just a good idea to make the types explicit